spec/feature/admin/admin_author_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'support/shared_context'
+
+RSpec.describe 'Admin_Author' do
```

Comment:
```suggestion
RSpec.describe 'Author' do
```
And rename file `/authors.spec`. The same for other admin pages tests